cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -1458,6 +1458,19 @@ class BufferMayWriteSideEffectInstruction extends SideEffectInstruction {
   }
 }
 
+/**
+ * 
```

Comment:
Brief comment please

Reply:
whoops, meant to fill that in before submitting